samples/algorithms/sudoku-grover/Program.cs

```diff
@@ -151,19 +164,19 @@ static void Main(string[] args)
                 bool resultFound = sudokuQuantum.QuantumSolve(puzzle9_2, sim).Result;
                 VerifyAndShowResult(resultFound, puzzle9_2, answer9);
             }
-            if (puzzleToRun == "9x9-64" || puzzleToRun == "all") 
+            if (puzzleToRun == "9x9-21" || puzzleToRun == "all") 
             {
                 // Test hard 9x9 puzzle with classical and quantum.
                 int[,] puzzle9 = {
-                    { 0,0,0, 0,0,0, 0,1,2 },
-                    { 0,0,0, 0,3,5, 0,0,0 },
-                    { 0,0,0, 6,0,0, 0,7,0 },
-                    { 7,0,0, 0,0,0, 3,0,0 },
-                    { 0,0,0, 4,0,0, 8,0,0 },
-                    { 1,0,0, 0,0,0, 0,0,0 },
-                    { 0,0,0, 1,2,0, 0,0,0 },
-                    { 0,8,0, 0,0,0, 0,4,0 },
-                    { 0,5,0, 0,0,0, 6,0,0 } };
+                    { 0,7,3, 8,0,0, 5,1,2 },
+                    { 9,0,2, 7,0,5, 4,8,6 },
+                    { 8,4,5, 0,0,0, 0,0,0 },
+                    { 7,0,8, 2,0,1, 3,5,0 },
+                    { 5,2,6, 4,0,3, 8,9,1 },
+                    { 1,3,4, 5,0,0, 0,0,0 },
+                    { 4,6,9, 1,2,8, 7,3,5 },
+                    { 2,8,7, 3,5,6, 1,4,9 },
+                    { 3,5,1, 9,4,7, 6,0,8} };
                 int[,] puzzle9_copy = CopyIntArray(puzzle9);
 
                 Console.WriteLine("Solving 9x9 with 64 missing numbers using classical computing.");
```

Comment:
Let's update the log message to match the update in the puzzle (both here and on line 187)